ipam/allocator.go

```diff
@@ -116,8 +116,10 @@ func (alloc *Allocator) Start() {
 // Stop makes the actor routine exit, for test purposes ONLY because any
 // calls after this is processed will hang. Async.
 func (alloc *Allocator) Stop() {
-	alloc.ticker.Stop()
-	alloc.actionChan <- nil
+	select {
```

Comment:
shouldn't you amend the comment to say something like "attempts to make the actor routine exit"?